tests/__fixtures__/search/searchIntents.fixtures.ts

```diff
@@ -91,7 +91,7 @@ export const mockSearchIntentsInitialState = {
 export const mockSearchIntentsState = {
   search: {
     intents: {
-      error: 'Error - Search request.',
+      error: { message: 'Error - Search request.' },
```

Comment:
Since we have a `mockSearchIntentsErrorState` do we even need this error to be anything? 🤔

Reply:
Yup, you're right! Applied ✅